handlers/couchbase/couchbase.go

```diff
@@ -0,0 +1,129 @@
+package couchbase
+
+import (
+	"errors"
+
+	"github.com/netflix/rend/common"
+	"github.com/netflix/rend/handlers"
+	gocb "gopkg.in/couchbase/gocb.v1"
+)
+
+var (
+	errNotFound = "key not found"
+)
+
+type Handler struct {
+	client clientWrapper
+}
+
+type clientWrapper interface {
+	set([]byte, []byte, uint32) error
+	get([]byte, *[]byte) error
+	close() error
+}
+
+type couchbaseClient struct {
+	client *gocb.Bucket
+}
+
+func (c *couchbaseClient) close() error {
+	return c.client.Close()
+}
+
+func (c *couchbaseClient) get(key []byte, data *[]byte) error {
+	_, err := c.client.Get(string(key), data)
```

Comment:
Those `string(key)` seems very inneficiant to me
Can you put this link in comment if we need it https://github.com/golang/go/issues/25484

Reply:
Done